src/_globalColor.scss

```diff
@@ -0,0 +1,87 @@
+/* Change colors in this file to style to your needs on a global level. 
+Certain colors are used for different and multiply types of elements. 
+For greater control over colors visit each components .scss file. */
+
+// button colors 
+$buttonColor: #55198b;
+$buttonHover: #8c43ce;
+$topButtonHover: #000;
+
+// text colors light theme 
+$titleColor:  #000000;
+$textColor: black;
+$subTitle: #868e96;
+$cardSubtitle: #666666;
+$talkCardSubTitle: #7f8287;
+$blogCardTitleColor: #262626;
+
+// text color dark theme 
+$textColorDark: white;
+
+// toggle switch colors
+$toggleCheck: #2196f3;
+$toggleSwitchSliderBG: #ccc;
+
+// githubRepo specific colors
+$githubRepoCardLanguageColorBG: blue;
+$githubRepoCardColor: rgb(88, 96, 105);
+$githubRepoCardRepoCardStatsColor: rgb(106, 115, 125);
+$githubRepoCardRepoNameColor: rgb(36, 41, 46);
+$githubProfileCardLocationTS: blanchedalmond;
+$githubProfileCardBorder: #6c63ff;
+
+// light background colors 
+$lightBackground1: #fff;
+$lightBackground2: rgb(255, 255, 255);
+$lightBackground3: #f5f2f4;
+$blogCardContainerColor: #586069;
+// dark background colors 
+$darkBackground: #171c28;
+
+// light theme box shadows
+$lightBoxShadow1: rgba(0, 0, 0, 0.2);
+$lightBoxShadow2: rgba(0, 0, 0, 0.1);
+$lightBoxShadow3: rgba(0, 0, 0, 0.3);
+$lightBoxShadow4: rgba(0, 0, 0, 0.08);
```

Comment:
It would nice to add a comment about where these colours are used  are used.

Reply:
We can rename these to:
 - `$lightBoxShadow1` -> `$lightBoxShadowDark` 
 - `$lightBoxShadow2` -> `$lightBoxShadow` 
 - `$lightBoxShadow3` -> `$lightBoxShadowDarker`
 - Remove `$lightBoxShadow4` because it is only being used in `Blog.scss` and can be replaced with current `$lightBoxShadow2` as the colors are very close shades of each other.
 
The advantage of renaming it this way would help people who want to use these variables in the future to design their own components without getting confused.